app/javascript/components/SignUp.jsx

```diff
@@ -1,4 +1,5 @@
 import React, { Component } from "react";
+import Tweetes from '../components/Tweetes';
```

Comment:
This component should be renamed to `Tweets` instead of `Tweetes`

Reply:
I put this name to make it a little bit different from the original tweets, from Twitter, as this website is called, retwitter, i changed the names of the things it contains, to make it different as well.

Reply:
That's not necessary. This name is confusing and it will get harder to follow in the future if we keep changing the names.